divided adminView to attribute, attribute_group, building_class and s…

vaisanem opened the pull request:
…ession

vaisanem committed:
divided adminView to attribute, attribute_group, building_class and session

xTooth commented on src/views/building_class.py:
```diff
@@ -0,0 +1,119 @@
+    return redirect(url_for("views.link_bclass_attribute_view", class_id = link.buildingclass_id))
+
+
+# Edit class attribute probability
```
this could use and error handler that checks for correct syntax.

xTooth commented on src/views/building_class.py:
```diff
@@ -0,0 +1,119 @@
+def edit_class_attribute_probability(class_attribute_id):
+    link = ClassAttribute.query.get(class_attribute_id)
+
+    try:
```
This doesnt seem to work correctly, requires further investigation

xTooth reviewed (approved):
this is a major change so a second pair of eyes might be good. I found no issues with the refactor itself, just unexpected behaviour in the original code.

xTooth committed:
Merge branch 'dev' into refactor-admin-view

xTooth committed:
Fixed a 500 error

MODIFIED
> building_class.py
 - added session rollbacks to fix 500 server errors.
